ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/PaymentMapper.java

```diff
@@ -57,8 +57,12 @@ public Payment from(CCDCase ccdCase) {
             ccdCase.getPaymentId(),
             moneyMapper.from(ccdCase.getPaymentAmount()),
             ccdCase.getPaymentReference(),
-            ccdCase.getPaymentDateCreated() != null ? ccdCase.getPaymentDateCreated().format(ISO_DATE) : null,
-            PaymentStatus.fromValue(ccdCase.getPaymentStatus()),
+            ccdCase.getPaymentDateCreated() != null
+                ? ccdCase.getPaymentDateCreated().format(ISO_DATE)
+                : null,
+            ccdCase.getPaymentStatus() != null
+                ? PaymentStatus.fromValue(ccdCase.getPaymentStatus())
```

Comment:
We have payment status as `Enum` in CMC domain but in CCD side we have this as a `String`. I think we should have this enum on both places.

Reply:
Yeah, I left it as string because we decided we didn't want to go through a data migration

Reply:
Let's discuss that and do that in separately, and merge this as it is to fix the issue in prod.